src/components/utilities/utilities.config.yml

```diff
@@ -1,8 +1,38 @@
+title: "Utilities"
 status: "ready"
 context:
-  demos:
-    - "One"
-    - "Two"
-    - "Three"
-    - "Four"
-    - "Five"
+    flexText:
+        - "One"
+        - "Two"
+        - "Three"
+        - "Four"
+        - "Five"
+    spacings:
```

Comment:
What do you think about creating another object in here similar to this `spacings` object called something like `typeScale` where we can store the values of the type scale. That way we can do something similar in the `utilities--type-scale.njk` component file where we loop through and create the markup for the example. That way the values could be update in one single place if needed instead of digging through the hard-coded markup. Does that make sense?

Reply:
Makes sense to me!

Reply:
Agreed! 👍

Reply:
Along with refactoring the Nunjucks and config.yml for `type-scale`, I've refactored each of the rest of the utilities as well.